### ISSUE TITLE
MetaModel Deprecation Warning: Conversion of an array with ndim > 0 to a scalar

### ISSUE THREAD

**Issue by BrianSchiller** [facebookresearch/nevergrad#1612]
## Steps to reproduce

  1. Create an optimizer that uses MetaModel, such as MetaModelOnePlusOne and give it a budget of ~50
  2. Run the model with a configuration that calls the function _learn_on_k_best_
  3.  _learn_on_k_best_ then throws DeprecationWarnings

## Observed Results

  * model_predict is called and returns an array which is then converted into a float value
  * In metamodel.py:83 there is usually only a single element in the returned array which shouldn't cause any problems
  * However in a later part of the code model.predict is called again and converted to float, however here there are multiple values that are returned which could lead to errors when converting to float (loss of information?)
 (I don't understand the inner working of the MetaModel enough to judge if this could lead to some problems with the performance)

## Expected Results

  * model.predict should only return a single value or the returned value shouldn't directly be converted to float
  * In general the deprecation warning is thrown a great amount of times and needs to be manually silenced (or fixed) in order for the terminal output to be readable

## Relevant Code
nevergrad\optimization\metamodel.py:83:
  ```
minimum = optimizer.minimize(
                        lambda x: float(model.predict(polynomial_features.fit_transform(x[None, :])))
                    ).value
  ```
(This part only has a single element being returned and therefor should work fine)

nevergrad\optimization\metamodel.py:91:
  ```
if float(model.predict(polynomial_features.fit_transform(minimum[None, :]))) > y[0]:
  ```
(This part has multiple values being returned)

The returned array can look like this:
```
[0.8339156  0.00786092 0.85692649 0.8821352  0.9505408  0.9708708
 0.11402067 0.98607812 1.00224294 0.98692334]

```

**Comment by teytaud** [comment]
Hello. Sorry for the delay.
This part of the code has moved a lot in the mean time as the MetaModel part has been improved significantly.
On the current version (github/main) I do not get any deprecation warning.
If you share the exact code with which you get this, I will double check.

**Comment by teytaud** [comment]
As seemingly the problem does not exist anymore I close the issue. In case you still meet a problem with the current github/main, please reopen + post the exact code and I'll investigate.
